### PR TITLE
Added Robert to the team

### PR DESCRIPTION
**Pull request by rjczanik**
Added a photo to the team content directory and description for Robert to team yaml file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ public
 resources
 .DS_Store
 .idea
+.vscode
```
Comment (bennycornelissen) [inline .gitignore]: I would normally agree with adding an exception for `.vscode` but in our case some relevant config that is used by Gitpod resides in that folder.

File: data/team.yml [hunk 2]
```diff
@@ -180,3 +180,16 @@ members:
 
 
       Aside from software, she enjoys recharging outdoors and discovering new hobbies, from cycling to dance classes.
+
+  - category: Engineering
+    image: "robert.jpg"
+    name: "Robert Czanik"
+    teaser: "Machine Learning Engineer | Full Stack Data Scientist | Braai master | Former astronomer "
+    description: >
+      Robert has a strong background in the hard sciences and is a seasoned data science professional. He is passionate about everything data related, from Postgres databasis, deep learning models to everything that build, deploys or shows the value in data.
```
Comment (bennycornelissen) [inline data/team.yml]: Did you actually mean 'databases', or is 'databasis' an actual thing? 😛

**Review by bennycornelissen — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (rjczanik) [reply .gitignore]: Removed

File: data/team.yml (hunk 2 above)
Comment (rjczanik) [reply data/team.yml]: Sticking to English, so it should be databases - and resolved.

**Review by bennycornelissen — APPROVED** [review]
LGTM 🎉

**Pull request merged**